src/chapters/cloudfront/cloudfront-fundamentals.js

```diff
@@ -0,0 +1,124 @@
+export const cloudfrontFundamentalsChapter = {
+  id: "cloudfront-fundamentals",
+  title: "CloudFront Fundamentals",
+  sectionId: "cloudfront",
+  previousChapterId: null,
+  content: `## Welcome Back to the Cloud
+
+In our last module, we took our first step into cloud development by deploying our Rock of Ages application to Amazon S3. You created a bucket, configured it for website hosting, and uploaded your React application files. By the end of that module, you had a working website accessible via your S3 bucket's website endpoint.
+
+Today, we're going to enhance your cloud deployment with Amazon CloudFront. This addition will significantly improve your application's performance, security, and global accessibility.
+
+## What We'll Accomplish Today
+
+By the end of this module, your Rock of Ages application will be:
+- Served through a global content delivery network
+- Available with HTTPS security
+- Optimized for speed with automatic file compression
+- Ready to handle visitors from anywhere in the world
+
+Let's start by understanding what CloudFront is and why it's valuable for your website.
+
+## What is CloudFront?
```

Comment:
Can we include a video overview here also? 

Found this video that does a good job explaining https://www.youtube.com/watch?v=OS5yTttzARE
The embed code for it is: 

```
<iframe width="560" height="315" src="https://www.youtube.com/embed/OS5yTttzARE?si=sJmWEFxpDO2CskOk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
```

Reply:
@gcziprusz Awesome! Looks great. What do you think about adding a message about the video saying to watch but not follow the tutorial? I don't want the students to be confused about what they're supposed to do here.

Reply:
That's great 👍